.github/workflows/test.yml

```
@@ -20,6 +20,9 @@
           distribution: 'corretto'
           cache: gradle
 
+      - name: Pull pgvector image
+        run: docker pull pgvector/pgvector:pg16
+
       - name:  Run tests
```

YAMLlint (1.37.1)

[error] too many spaces after colon

(colons)